app/scripts/constants/contracts.ts

```diff
@@ -11,4 +11,5 @@ export const SINGLE_CALL_BALANCES_ADDRESSES = {
   [CHAIN_IDS.AVALANCHE]: '0xD023D153a0DFa485130ECFdE2FAA7e612EF94818',
   [CHAIN_IDS.FANTOM]: '0x07f697424ABe762bB808c109860c04eA488ff92B',
   [CHAIN_IDS.ARBITRUM]: '0x151E24A486D7258dd7C33Fb67E4bB01919B7B32c',
+  [CHAIN_IDS.BLAST]: '0xfd5730e96f9dffae40d99b77015bd42816280998',
```

Comment:
Future TODO: the bulk balance contract is available on more chains than this list has.  Will add more in a separate PR.